README.md

```diff
@@ -1102,3 +1116,24 @@ Check out the demo in the folder `/manifests/regex-parser` for how to use a cust
 ### Manifests
 
 Most files under the folder [manifests/setup](manifests/setup) are automatically generated from [config](config). Don't edit them directly, run `make manifests` instead, then replace these files accordingly.
+
+## Community
+
+### About the community
+
+Fluent Operator is sponsored and open-sourced by the [KubeSphere](http://kubesphere.io/) Team and maintained by the Fluent community. This project was accepted by the Fluent community as a sub-project in 2021 and is currently a vendor-neutral and community-driven project. 
```

Comment:
Split sentences into multiple lines as easier to review.

Reply:
Updated.